components/DetailPageSidebar/detail-page-sidebar.scss

```diff
@@ -0,0 +1,131 @@
+@import '~carbon-components/scss/globals/scss/styles.scss';
+
+@include exports('detail-page-sidebar') {
+  .bx--detail-page-sidebar__container { 
+    width: 100%;
+    margin: 0 5% 0 0; 
+  }
+  .bx--detail-page-sidebar__container .tag-description,
+  .bx--detail-page-sidebar__container .details-description {
+      font-size: 0.875rem;
+      line-height: 1.65;
+      font-weight: 400;
+      word-wrap: break-word; 
+  }
+  .bx--detail-page-sidebar__container .tag-description {
+      margin-bottom: 2em; 
+    }
+  .bx--detail-page-sidebar__container .tag-description > span {
+        font-weight: 600; 
+  }
+    @media (max-width: 824.999px) {
```

Comment:
Additionally, what is specifying this breakpoint? Carbon has a couple set ones out of the box, this is quite.. an odd number